packages/kbn-config-schema/src/types/buffer_type.test.ts

```diff
@@ -49,8 +53,4 @@ test('returns error when not a buffer', () => {
   expect(() => schema.buffer().validate([1, 2, 3])).toThrowErrorMatchingInlineSnapshot(
     `"expected value of type [Buffer] but got [Array]"`
   );
-
-  expect(() => schema.buffer().validate('abc')).toThrowErrorMatchingInlineSnapshot(
-    `"expected value of type [Buffer] but got [string]"`
-  );
```

Comment:
`joi.binary` now converts strings to buffers (using the default encoding)

I thought it was fine (at least it didn't break anything), but if we want to forbid that, I can add our own `coerce` function for the `binary` type